src/static/css/video-player.less

```diff
@@ -0,0 +1,225 @@
+
+/* ==========================================================================
+   cfgov-refresh
+   video-player
+   ========================================================================== */
+
+/* topdoc
+  name: Video Player
+  family: cfgov-video-player
+  patterns:
+      notes:
+        - "For the markup please see macros/video-player.html."
+  tags:
+    - cfgov-video-player
+*/
+
+// Variable used to set the size of the play button
+@play-btn-size__px: 73px;
+
+// Variable used to set the play button image
+@play-btn-url: '/static/img/video-play-btn.png';
+
+
+// Variable used to help center the video container
+@share-width__px: 130px;
+
+// Variables used for gutters
+@gutter__em: unit(@grid_gutter-width  / @base-font-size-px, em);
+@gutter-half__em: @gutter__em / 2;
+
+.video-player {
+    overflow: hidden;
+    background: @white;
+}
+
+.video-player.video-playing {
+    background-color: #000;
+
+    .hide-on_video-playing {
+        display: none;
+    }
+
+    .show-on_video-playing {
+        display: block;
+    }
+
+    .respond-to-max(@mobile-max, {
+        margin-top: 0;
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/38a787781f8ef2dcd1d5aaebedd239e3a388d9cd